src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,31 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type CommentProps = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<CommentProps> = ({ comment }) => {
```

Comment:
Generally mate-academy suggests using React.FC however I personally don't agree with that typing as it unnecessary adds children to props, this might create confusing components that shouldn't receive children (as they don't do anything with them) but can, more on that in this pr to official React repo 
-> https://github.com/facebook/create-react-app/pull/8177

Reply:
I added types in props parameters, because it is shorter.